### PR TITLE
Display astronauts on Homepage

### PR DESCRIPTION
**Pull request by mudasiri**
In this Pull request I implemented a call to the API we are working with by calling the list of astronauts and showing them on the homepage. Each card for an astronauts displays the following information:

- Full name
- Image
- Love/Like icon
- Comments button
- Reservation Button.

@jash-debug  kindly review so I can proceed to next Task.

### CODE REVIEW

**Comment by jash-debug** [comment]
Hi @mudasiri ,
Good job so far the project look great 👍🏼
CatDrivingSeriousGIF


In js:
- [ ] kindly use the alt tag for image files
https://github.com/mudasiri/javascript-capstone/blob/b5a7418ad6585c130dad0c2d91127fd3c4fb800f/src/modules/helper.js#L9

👍 linter `errors`
✔️ reviewed project run on your machine
✔️ Used best practices
✔️ Used Github Flow
✔️ Work documented professionally
🎟️ Full Functional site

Please, as recommended among the [best practices for HTML & CSS](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md), kindly avoid the unnecessary use of blank lines in your code. This offers better readability to your code. This has many occurrences in your code.

**Comment by mudasiri** [comment]
Thanks a lot for the review I will fix Asap and Alert

**Comment by mudasiri** [comment]
@jash-debug  I have added the Alt text and made it dynamic in the format of {astronautname-image}

**Comment by jash-debug** [comment]
Hi @mudasiri  ,
Good job so far the project look great 👍🏼, You can merge the project

GoodGoodClappingHandsGIF


👍 linter errors
✔️ reviewed project run on your machine
✔️ Used best practices
✔️ Used Github Flow
✔️ Work documented professionally
🎟️ Full Functional site

Please, as recommended among the [best practices for HTML & CSS](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md), kindly avoid the unnecessary use of blank lines in your code. This offers better readability to your code. This has many occurrences in your code.

File: src/modules/helper.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+// grab main tag displaying astronauts items
+const astronautList = document.getElementById('astronaut');
+
+// create arrow function to loop through and display austraunauts lists from API
+export const displayAstronauts = (astronauts) => {
+    astronautList.innerHTML = '';
+    astronauts.forEach((astronaut) => {
+      // append div to main tag
+      astronautList.insertAdjacentHTML('beforeend', `<div class='person'><img class='person-img' src= '${astronaut.image}'> <div class='title-area'><h2>${astronaut.name}</h2> <i class="heart fa-regular fa-heart"></i> </div> <button>Comments</button><button>Reservations</button></div>`);
```
Comment (jash-debug) [inline src/modules/helper.js]: You need to add an alt name for the image being loaded from the Api

**Pull request merged**